include/ts/apidefs.h.in

```diff
@@ -501,6 +515,29 @@ typedef enum {
 } TSEvent;
 #define TS_EVENT_HTTP_READ_REQUEST_PRE_REMAP TS_EVENT_HTTP_PRE_REMAP /* backwards compat */
 
+#ifdef __cplusplus
```

Comment:
Better, but these should be moved to a ".cc" file, both to clean up the header and to avoid having to check for a C++ compiler.

Reply:
Sounds good. Thanks.